assets.sh

```diff
@@ -31,47 +34,61 @@ NCN_ARCH='x86_64'
 CN_ARCH=("x86_64")
 
 # All images must use the same, exact kernel version.
-KERNEL_VERSION='6.4.0-150600.23.17-default'
-# NOTE: The kernel-default-debuginfo package version needs to be aligned
-# to the KERNEL_VERSION. Always verify and update the correct version of
-# the kernel-default-debuginfo package when changing the KERNEL_VERSION
-# by doing a zypper search for the corresponding kernel-default-debuginfo package
-# in the SLE-Module-Basesystem update_debug repo
-# zypper --plus-repo=https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Basesystem/15-SP4/x86_64/update_debug se -s kernel-default-debuginfo
-KERNEL_DEFAULT_DEBUGINFO_VERSION="${KERNEL_VERSION//-default/}.1"
+KERNEL_VERSION='6.4.0-*-default'
 
 # The image ID may not always match the other images and should be defined individually.
-KUBERNETES_IMAGE_ID=6.2.33
+KUBERNETES_IMAGE_ID=6.2.*
+
+# Resolve globs in KUBERNETES_IMAGE_ID, e.g. 6.2.* > 6.2.30
+KUBERNETES_IMAGE_ID=$(basename $(dirname $(resolve_globs "csm-images" "stable/kubernetes/${KUBERNETES_IMAGE_ID}" "kubernetes-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.squashfs")))
+# Resolve globs in KERNEL_VERSION, e.g. 6.4.0-*-default > 6.4.0-150600.23.17-default
+KERNEL_PATH=$(resolve_globs "csm-images" "stable/kubernetes/${KUBERNETES_IMAGE_ID}" "${KERNEL_VERSION}-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.kernel")
+KERNEL_VERSION=$(basename "${KERNEL_PATH}")
+KERNEL_VERSION=${KERNEL_VERSION%-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.kernel}
 KUBERNETES_ASSETS=(
     "https://artifactory.algol60.net/artifactory/csm-images/stable/kubernetes/${KUBERNETES_IMAGE_ID}/kubernetes-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.squashfs"
     "https://artifactory.algol60.net/artifactory/csm-images/stable/kubernetes/${KUBERNETES_IMAGE_ID}/${KERNEL_VERSION}-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.kernel"
     "https://artifactory.algol60.net/artifactory/csm-images/stable/kubernetes/${KUBERNETES_IMAGE_ID}/initrd.img-${KUBERNETES_IMAGE_ID}-${NCN_ARCH}.xz"
 )
 
 # The image ID may not always match the other images and should be defined individually.
-PIT_IMAGE_ID=6.2.33
+PIT_IMAGE_ID=6.2.*
+# Resolve globs in PIT_IMAGE_ID, e.g. 6.2.* > 6.2.30
+PIT_IMAGE_ID=$(basename $(dirname $(resolve_globs "csm-images" "stable/pre-install-toolkit/${PIT_IMAGE_ID}" "pre-install-toolkit-${PIT_IMAGE_ID}-${NCN_ARCH}.iso")))
 PIT_ASSETS=(
     "https://artifactory.algol60.net/artifactory/csm-images/stable/pre-install-toolkit/${PIT_IMAGE_ID}/pre-install-toolkit-${PIT_IMAGE_ID}-${NCN_ARCH}.iso"
 )
 
 # The image ID may not always match the other images and should be defined individually.
-STORAGE_CEPH_IMAGE_ID=6.2.33
+STORAGE_CEPH_IMAGE_ID=6.2.*
```

Comment:
Temporary change to test glob processing. Represents a sample change which we may have in integration branch. Will be reverted before merge into release branch.